tests/integration/ovs_test.py

```diff
@@ -0,0 +1,133 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import yaml
+
+from libnmstate import netapplier
+
+from .testlib import statelib
+from .testlib.statelib import INTERFACES
+
+
+OVS_BRIDGE_YAML_BASE = """
+interfaces:
+  - name: ovs-br0
+    type: ovs-bridge
+    state: up
+    bridge:
+      options:
+        fail-mode: ''
+        mcast-snooping-enable: false
+        rstp: false
+        stp: true
+"""
+
+
+def test_create_and_remove_ovs_bridge_with_a_system_port(eth1_up):
```

Comment:
Usually you are using contextmanagers to create and remove the virtual test interfaces, why not here and in the other OVS tests?

Reply:
Not due to good reasons. I started it by copying the Linux bridge tests.
I will pass over all of these in a following PR and fix it.